decidim-core/app/helpers/decidim/widget_urls_helper.rb

```diff
@@ -2,8 +2,8 @@
 module Decidim
   module WidgetUrlsHelper
     def embed_modal_for(url)
-      js_embed_code = "#{content_tag(:script, '', src: url)}"
-      embed_code = "#{content_tag(:noscript, content_tag(:iframe, '', src: url.gsub(".js", ".html"), frameborder: 0, scrolling: "vertical"))}"
+      js_embed_code = content_tag(:script, "", src: url).to_s
```

Comment:
Sure, sounds good! I'm just not sure of the wording.

By the way, `String.new(content_tag(...))` works as well, and it blows up if `to_str` is not defined or does not return a `String` instance. So maybe it's a bit more resilient than `to_str`?

```
[1] pry(main)> class A
[1] pry(main)*   def to_str
[1] pry(main)*     "hi"
[1] pry(main)*   end  
[1] pry(main)* end  
=> :to_str
[2] pry(main)> class B
[2] pry(main)*   def to_str
[2] pry(main)*     self
[2] pry(main)*   end  
[2] pry(main)* end  
=> :to_str
[3] pry(main)> class C; end
=> nil
[4] pry(main)> String.new(A.new)
=> "hi"
[5] pry(main)> String.new(B.new)
TypeError: can't convert B to String (B#to_str gives B)
from (pry):13:in `initialize'
[6] pry(main)> String.new(C.new)
TypeError: no implicit conversion of C into String
from (pry):14:in `initialize'
```

The lesson here, I think, is that `to_s` seems to be good for "object representation" and `to_str` for "type conversion", and we want the latter here.

Sorry for disgressing this much, at least we are learning Ruby :)